balancer-js/src/modules/relayer/relayer.module.ts

```diff
@@ -99,6 +100,18 @@ export class Relayer {
     ]);
   }
 
+  // TODO: check if it's ok to import Swap from swaps module - alternatives: move to common module or duplicate within relayer types
```

Comment:
I think this is fine for now. We def don't want to duplicate. In an ideal world I think we'd share a lot more of the logic in an easy way but its probably too big a refactor for now.